templates/compose/pi-hole.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  pihole:
+    image: 'pihole/pihole:latest'
+    ports:
+      - '53:53/tcp'
+      - '53:53/udp'
+      - '8080:80'
```

Comment:
Should be removed, and instead, let reverse proxy handle it.

add the key `SERVICE_FQDN_PIHOLE_81` in environment, and coolify will automatically handle it.

example:
```yaml
environment:
  - SERVICE_FQDN_PIHOLE_81
```

Reply:
I agree. Line 7 should be removed.

It is also safer to not expose the application port directly. Traefik can put some security in front of it if someone needs it.